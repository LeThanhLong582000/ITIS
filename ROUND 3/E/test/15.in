17
1777 9249
9161 4550
2761 6641
4501 6709
273 1911
2677 2991
1616 1025
489 6513
151 8675
9953 4565
708 6821
7341 9519
7729 1331
7201 5615
8101 5857
4045 4171
1791 5865
