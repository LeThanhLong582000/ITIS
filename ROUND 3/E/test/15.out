270124624946768
1737434599524066
336201433402200
911871159749304
272169351074
64110487712486
2743646084076
10143285530744
1715891822196
2064379253603220
23321786042744
4883071077373934
105828428238414
1634876858219466
2251270238407716
284653940426346
110338438320146
